core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.component.ts

```diff
@@ -78,6 +79,11 @@ interface ActiveContributionDict {
   'suggestion': ActiveSuggestionDict;
 }
 
+enum ExpansionTabType {
+  'content' = 'content',
+  'translation' = 'translation'
```

Comment:
Prefer to declare as numeric enum and in all caps to make usage more like a "constant".

```suggestion
  CONTENT,
  TRANSLATION
```

Reply:
Done